curvefs/src/client/s3/client_s3_cache_manager.h

```diff
@@ -91,25 +91,32 @@ struct ObjectChunkInfo {
 class DataCache {
  public:
     DataCache(S3ClientAdaptorImpl *s3ClientAdaptor,
-              ChunkCacheManager *chunkCacheManager, uint64_t chunkPos,
-              uint64_t len, const char *data)
+              const std::shared_ptr<ChunkCacheManager>& chunkCacheManager,
+              uint64_t chunkIndex, uint64_t chunkPos, uint64_t len,
+              const char *data)
         : s3ClientAdaptor_(s3ClientAdaptor),
-          chunkCacheManager_(chunkCacheManager), chunkPos_(chunkPos), len_(len),
-          dirty_(true) {
+          chunkCacheManager_(chunkCacheManager),
+          chunkIndex_(chunkIndex),
+          chunkPos_(chunkPos),
+          len_(len),
+          dirty_(true),
+          inReadCache_(false) {
         data_ = new char[len];
         memcpy(data_, data, len);
         createTime_ = ::curve::common::TimeUtility::GetTimeofDaySec();
-        //  dirty_.exchange(true, std::memory_order_acq_rel);
     }
     DataCache(S3ClientAdaptorImpl *s3ClientAdaptor,
-              ChunkCacheManager *chunkCacheManager, uint64_t chunkPos,
-              uint64_t len)
+              const std::shared_ptr<ChunkCacheManager>& chunkCacheManager,
```

Comment:
& is near chunkCacheManager，The format needs to be uniform

Reply:
fix